question28/js/validations.js

```diff
@@ -0,0 +1,80 @@
+const submitButton = document.getElementById('js-submit-button');
+
```

Comment:
IMO
I think it is better to put the files to be treated as modules in the module directory.

Reply:
Thank you for your review!
I thought you are right. Created modules directory to contain validation.js.

fixed commit : https://github.com/csakyo/homework/pull/42/commits/47ba8a4db2bed2e90dc90588565b066b66b76985